THER20Q3

```diff
@@ -0,0 +1,10001 @@
+primaryid$caseid$dsg_drug_seq$start_dt$end_dt$dur$dur_cod
+100046943$10004694$1$20100501$20120213$NA$
+100046943$10004694$2$20100728$20120213$NA$
+100046943$10004694$3$20121009$20121213$NA$
+100046943$10004694$4$20120316$20121213$NA$
+100046943$10004694$5$20091231$20100130$NA$
+100049343$10004934$1$200507$2011$NA$
+100049343$10004934$4$2011$201707$NA$
+100051353$10005135$1$20031222$2008$NA$
+100051353$10005135$2$2006$2008$NA$
```

Comment:
Non metterei file così grandi, per favore riportali tutti a 10 righe di dati (più l'intestazione), o anche meno se vuoi. piuttosto fai più file, in cui per esempio in uno metti un tipo di contenuto (per esempio un numero intero o vuoto) e nell'altro file metti un altro tipo di dato (rispettivamente, per esempio, un numero decimale o un carattere), e controlli che entramabi i file siano importati correttamente (corrispondentemente, nell'esempio, come double o character, invece che integer e logical).

Questo vale per tutti i file di test.